_includes/config/es-elasticsearch-magento.md

```diff
@@ -1,102 +1,98 @@
-	In particular, make sure you started Elasticsearch as a user with <code>root</code> privileges.</li>
-<li>Make sure that <a href="{{ site.gdeurl21 }}config-guide/elasticsearch/es-overview.html#firewall-selinux">UNIX firewall and SELinux</a> are both disabled, or set up rules to enable Elasticsearch and Magento to communicate with each other.</li>
-	<li>Verify the value of the <strong>Elasticsearch Server Hostname</strong> field. Make sure the server is available. You can try the server's IP address instead.</li>
-	<li>Use the command <code>netstat -an | grep <em>listen-port</em></code> command to verify that the port specified in the <strong>Elasticsearch Server Port</strong> field is not being used by another process.<br />
-	For example, to see if Elasticsearch is running on its default port, use the following command:
-	<pre>netstat -an | grep 9200</pre>
-	If Elasticsearch is running on port 9200, it displays similar to the following:
-	<pre>tcp        0      0 :::9200            :::*          LISTEN</pre></li></ul></td>
-	</tr>
-</tbody>
-</table>
-
-### Reindexing catalog search and refreshing the full page cache {#es-reindex}
+1. Log in to the Magento Admin as an administrator.
+1. Click **Stores** > Settings > **Configuration** > **Catalog** > **Catalog** > **Catalog Search**.
+1. From the **Search Engine** list, select the correct Elasticsearch version as the following figure shows.
+   (The **Elasticsearch 5.0+/6.0+** options are not available for Magento 2.1.)
+
+![]({{ site.baseurl }}/common/images/elastic_choose-in-admin-es6.png){:width="650px"}
+
+The following table lists the required configuration options to configure and test the connection with Magento.
+Unless you changed Elasticsearch server settings, the defaults should work. Skip to the next step.
+
+|Option|Description|
+|--- |--- |
+|**Elasticsearch** Server Hostname|Enter the fully qualified hostname or IP address of the machine running Elasticsearch. {{site.data.var.ece}}: Get this value from your integration system.|
+|**Elasticsearch Server Port**|Enter the Elasticsearch web server proxy port. In our example, the port is 8080 but if you are using a secure proxy, it is typically 443. {{site.data.var.ece}}: Get this value from your integration system.|
+|**Elasticsearch Index Prefix**|Enter the Elasticsearch index prefix. If you use a single Elasticsearch instance for more than one Magento installation (Staging and Production environments), you must specify a unique prefix for each installation. Otherwise, you can use the default prefix magento2.|
+|**Enable Elasticsearch HTTP Auth**|Click **Yes** only if you enabled authentication for your Elasticsearch server. If so, provide a username and password in the provided fields.|
+{:style="table-layout:auto;"}
+
+{:start="4"}
+1. Click **Test Connection**.
+
+You will see either:
```

Comment:
Just a question, see either meaning the image and/or the sentence Magento successfully connected to the Elasticsearch server? If just the image, change to You will see a success message: or something like that.
If image or text, then add a blank line before Continue with:

Reply:
The either refers to either of the two image below: success or not.